packages/core/src/FrameLoop.ts

```diff
@@ -62,11 +64,18 @@ export class FrameLoop {
 
     this.update =
       (update && update.bind(this)) ||
-      (() => {
+      ((time?: number) => {
         if (this.idle) {
           return false
         }
 
+        time = time !== void 0 ? time : Date.now()
```

Comment:
Use `performance.now` instead, since that's what rAF gives us.

Reply:
Done!